packages/lesswrong/components/common/Header.tsx

```diff
@@ -154,6 +165,208 @@ const styles = (theme: ThemeType): JssStyles => ({
       position: "fixed !important",
     },
   },
+  
+  // Below are custom styles for the special EA Forum Giving Season 2023 homepage header
+  // TODO: delete after 2023
```

Comment:
This isn't my favorite way to handle styling, but this was the fastest way to do it and I figured that we'd probably just want to remove the code after giving season

Reply:
Oh lawd

Reply:
Don't let any of the new devs see you do this.